doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -1805,3 +1849,184 @@ Example of HTTP2 logging, of a request and response:
       ]
     }
   }
+
+
+Event type: IKE
+---------------
+
```

Comment:
I checked SV tests and the outputs, and IKE and DHCP appear there. Didn't check for each independent field, though...

Reply:
someone will need to check :)

Reply:
Roger that 🙇🏽